pkg/interface/src/views/apps/launch/App.tsx

```diff
@@ -28,73 +27,85 @@ interface LaunchAppProps {
+      <Route path="/join/:ship/:name">
+        <JoinRoute modal />
       </Route>
-      <ScrollbarLessBox height='100%' overflowY='scroll' display="flex" flexDirection="column">
+      <ScrollbarLessBox
+        height="100%"
+        overflowY="scroll"
+        display="flex"
+        flexDirection="column"
+      >
         <Box
           mx={2}
-          display='grid'
-          gridTemplateColumns='repeat(auto-fill, minmax(128px, 1fr))'
+          display="grid"
+          gridTemplateColumns="repeat(auto-fill, minmax(128px, 1fr))"
           gridGap={3}
           p={2}
           pt={0}
         >
-        {!hideUtilities && <>
-          <Tile
-            bg="white"
-            color="scales.black20"
-            to="/~landscape/home"
-            p={0}
-          >
-            <Box
-              p={2}
-              height='100%'
-              width='100%'
-              bg='scales.black20'
-              border={1}
-              borderColor="lightGray"
-            >
-              <Row alignItems='center'>
-                <Icon
-                  color="black"
-                  icon="Home"
-                />
-                <Text ml={2} mt='1px' color="black">My Channels</Text>
-              </Row>
-            </Box>
-          </Tile>
-          <Tiles />
-          <ModalButton
-            icon="Plus"
-            bg="washedGray"
-            color="black"
-            text="New Group"
-            style={{ gridColumnStart: 1 }}
-          >
-            <NewGroup />
-          </ModalButton>
-          <ModalButton
-            icon="BootNode"
-            bg="washedGray"
-            color="black"
-            text="Join Group"
-          >
-            {dismiss => <JoinGroup dismiss={dismiss} />}
-          </ModalButton>
-          </>}
-          {!hideGroups &&
-            (<Groups />)
-          }
+          {!hideUtilities && (
+            <>
+              <Tile
+                bg="white"
+                color="scales.black20"
+                to="/~landscape/home"
+                p={0}
+              >
+                <Box
+                  p={2}
+                  height="100%"
+                  width="100%"
+                  bg="scales.black20"
+                  border={1}
+                  borderColor="lightGray"
+                >
+                  <Row alignItems="center">
+                    <Icon color="black" icon="Home" />
+                    <Text ml={2} mt="1px" color="black">
+                      My Channels
+                    </Text>
+                  </Row>
+                </Box>
+              </Tile>
+              <Tiles />
+              <ModalButton
+                icon="Plus"
+                bg="washedGray"
+                color="black"
+                text="New Group"
+                style={{ gridColumnStart: 1 }}
+              >
+                <NewGroup />
+              </ModalButton>
+              <Button
+                bg="washedGray"
```

Comment:
```suggestion
                backgroundColor="washedGray"
```

This button was invisible. I think the `bg` prop is special to `ModalButton` maybe?